src/js/components/help/connecting-devices/raspberry-pi-3.js

```diff
@@ -92,6 +92,9 @@ var RaspberryPi =  createReactClass({
         <p>Specifically, your device needs to be able to create outgoing TCP connections to hosted.mender.io and s3.amazonaws.com, both on port 443 (TLS).</p>
         <p>As the Mender client does not listen to any ports there are no incoming connections.</p>
 
+        <h4>Enabling wifi in the image</h4>
+        <p>Enabling wifi can be done, by following the instructions in: <a>https://docs.mender.io/development/getting-started/deploy-to-physical-devices</a></p>
```

Comment:
* Would remove "development" from the URL, which maps to the latest stable instead
* Would also link directly to the section in the docs "Wifi connectivitiy", after https://github.com/mendersoftware/mender-docs/pull/409 is merged
* They also need to get the mender-artifact utility, but suppose that is ok given it is on the same page. Or change to first get mender-artifact, then follow the Wifi connectivity doc.

Reply:
mender-artifact cp is still not in any released version.

Reply:
Ahh, right. Then keep development and make a reminder to remove it once 1.6 is out :)

Reply:
done

Reply:
github should have triggered reminders